### ISSUE TITLE
GitHub link doesn't show on Mobile, tech stack not shown

### ISSUE THREAD

**Issue by cdilga** [check-face/checkface#3]
Mobile doesn't have the github link

Also Nvidia and AWS etc aren't shown

**Comment by olivercoad** [comment]
What do you mean by this? They aren't on the landing page on desktop either, let alone on mobile.
 > Also Nvidia and AWS etc aren't shown

**Comment by cdilga** [comment]
Since we are pitching to these organisations, signalling the use of their services would be good. It needs to be both on desktop and mobile

**Comment by olivercoad** [comment]
Yeah absolutely. I was confused because it sounded like you were saying they just weren't shown on mobile, like the GitHub link.

**Comment by cdilga** [comment]
I've slightly progressed, as I now have an architecture diagram which we could add, though it's not super aesthetic and it's a little too AWS specific 
![Checkface Architecture Light Background](https://user-images.githubusercontent.com/4075815/62936867-36387500-be0e-11e9-94b2-7c48accb4aa6.png)
